warp10/src/main/java/io/warp10/script/processing/image/Presize.java

```diff
@@ -77,7 +79,13 @@ public Object apply(WarpScriptStack stack) throws WarpScriptException {
 
     if (top instanceof PImage) { // PGraphics extends PImage
       PImage img = (PImage) top;
+      if (top instanceof PGraphics) {
```

Comment:
You indeed found an issue with Presize on a Pgraphics but it seems this fix it not enough. 
```
1000 1000 '2D8' PGraphics
200 200 Presize
```
Produces
`sun.java2d.SunGraphics2D cannot be cast to java.awt.image.BufferedImage`